### ISSUE TITLE
[FR] rpz extended statistic support

### ISSUE THREAD

**Issue by raunz** [letsencrypt/unbound_exporter#69]
Small feature request as the current code isn't able to export [Response Policy Zones](https://unbound.docs.nlnetlabs.nl/en/latest/topics/filtering/rpz.html) related counters. Unbound has been RPZ aware for some time and we'd like to get these metrics from the unbound_exporter too.

unbound conf:
```
rpz:
        name: "rpz"
```
        
Extended statistics explantion:        
```
num.rpz.action.<rpz_action>
 Number of queries answered using configured RPZ policy, per RPZ action type. 
 Possible actions are: nxdomain, nodata, passthru, drop, tcp-only, local-data, disabled, and cname-override.
```
              
Example output from unbound-control stats (the name in my example is also rpz):
```
num.rpz.action.rpz-nxdomain=79524
num.rpz.action.rpz-local-data=2261
...
```

**Comment by pgporada** [comment]
Good idea, can do.

**Comment by raunz** [comment]
I fiddled around a bit with the source and got this working:
```
diff --git a/unbound_exporter.go b/unbound_exporter.go
index c104e2e..8aa8a52 100644
--- a/unbound_exporter.go
+++ b/unbound_exporter.go
@@ -320,6 +320,12 @@ var (
                        prometheus.GaugeValue,
                        nil,
                        "^rrset\\.cache\\.count$"),
+               newUnboundMetric(
+                       "rpz_action_count",
+                       "Total number of applied RPZ actions",
+                       prometheus.CounterValue,
+                       []string{"action"},
+                       "^num\\.rpz\\.action\\.rpz-([\\w-]+)$"),
        }
 )
```
I found that "rpz-" before action name is static string, not zone or RPZ name. https://github.com/NLnetLabs/unbound/blob/master/services/rpz.c#L68-L83 
Giving result:
```
# HELP unbound_rpz_action_count Total number of applied RPZ actions
# TYPE unbound_rpz_action_count counter
unbound_rpz_action_count{action="local-data"} 1
unbound_rpz_action_count{action="nxdomain"} 1
```
Shortened action name looks better than action="rpz-local-data"

**Comment by pgporada** [comment]
You should check out the PR I put up [here](https://github.com/letsencrypt/unbound_exporter/pull/70) :)

**Comment by pgporada** [comment]
@raunz I've published tag v0.4.5 that includes this change.
